test/kubemark/start-kubemark.sh

```diff
@@ -14,15 +14,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Script that creates a Kubemark cluster with Master running on GCE.
+# Script that creates a Kubemark cluster for a given cloud provider.
 
-# Hack to make it work for OS X. Ugh...
 TMP_ROOT="$(dirname "${BASH_SOURCE}")/../.."
 KUBE_ROOT=$(readlink -e ${TMP_ROOT} 2> /dev/null || perl -MCwd -e 'print Cwd::abs_path shift' ${TMP_ROOT})
+KUBECTL="${KUBE_ROOT}/cluster/kubectl.sh"
+KUBEMARK_DIRECTORY="${KUBE_ROOT}/test/kubemark"
+RESOURCE_DIRECTORY="${KUBEMARK_DIRECTORY}/resources"
+
+source "${KUBE_ROOT}/test/kubemark/skeleton/util.sh"
```

Comment:
Well I followed it similar to kube-up. Importing the skeleton first and then the provider-specific util would make it easily visible to the developer that a particular function of the interface wasn't defined in the util.